### DISCUSSION TITLE
Home Assistant OS 11.5.rc2

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3117] [Announcements]
Changes since 11.5.rc1

## Open Virtual Appliance

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon

## Generic x86-64

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon

## Hardkernel ODROID

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon

## ASUS Tinker

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon

## Khadas VIM Series

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.5.rc2'>Home Assistant OS 11.5.rc2</a>.</em>

**Comment by apbarratt** [comment]
Installed and working perfectly on Pi5 8GB with Pimoroni NVMe base and Crucial 2TB SSD as boot device.  All add-ons working as expected including Frigate.

**Reply by timchans** [reply]
I flash WD NVMe SSD via BalenaEtcher and insert to Pineberry hat but cannot boot up. Do I miss something?

**Reply by apbarratt** [reply]
First boot your Pi using an SD card with latest Pi-OS installed (latest preferably).  I'd use the official Raspberry Pi Image Flasher as it makes things super easy.  Once you have that installed and booted, at the terminal run `sudo raspi-config` then go to "Advanced" and "Boot-Order" to update the EEPROM to tell it to boot from the SSD first :)
I will however warn you that Pimoroni's website listed a few drives they couldn't get to work and they said WD drives especially were problematic (something to do with a SANDISK controller on them not playing nice).  They did have a few that worked but they said some were either poorly performing or simply refused to boot at all.  Here's the link: https://shop.pimoroni.com/products/nvme-base?variant=41219587178579. It doesn't list everything, but it's what led me to buy the Crucial drive.

**Reply by timchans** [reply]
Thanks a lot. I can boot via ssd after replacing the WD ssd with the crucial P3 plus. However, I cannot restore the home assistant and add Matter devices.

**Reply by apbarratt** [reply]
That's curious, restore worked flawlessly for me (actually first time a migration didn't encounter a hiccup at the restore stage for me!)

I'm using Zigbee with a SkyConnect so I can't comment on Matter I'm afraid.

**Comment by apbarratt** [comment]
Now with a pretty home to live in thanks to this excellent design: https://www.thingiverse.com/thing:6431038

![image](https://github.com/home-assistant/operating-system/assets/642320/231c6353-f061-4ea1-b09f-2104a60300dd)
![image](https://github.com/home-assistant/operating-system/assets/642320/e3120b10-b728-4da2-b118-beb77a8fa9d0)
![image](https://github.com/home-assistant/operating-system/assets/642320/ae0efdeb-5c7e-4bb5-a103-c1adcd66b14b)
